db/migrate/20200221200855_add_ancestry_to_miq_ae_namespace.rb

```diff
@@ -0,0 +1,31 @@
+require 'ancestry'
+
+class AddAncestryToMiqAeNamespace < ActiveRecord::Migration[5.1]
+  class MiqAeNamespace < ActiveRecord::Base
+    has_ancestry
+  end
+
+  def up
+    add_column :miq_ae_namespaces, :ancestry, :string
+    add_index :miq_ae_namespaces, :ancestry
+
+    say_with_time("Converting MiqAeNamespaces from parent_id to ancestry") do
+      MiqAeNamespace.build_ancestry_from_parent_ids!
```

Comment:
Is there any way to scope this to only the current region?

Reply:
nevermind, it's excluded.  carry on

Reply:
I believe that method is scope aware, so it would be worth it to try.